environments/linux-x86-wine_osxcross

```diff
@@ -27,7 +27,7 @@ RUN osxcross-macports -universal fakeinstall -v pulseaudio
 RUN env osxcross-macports -universal fakeinstall ncurses
 
 RUN osxcross-macports -universal install -v wine-devel
-RUN osxcross-macports -universal install gnutls
+# RUN osxcross-macports -universal install gnutls
```

Comment:
Will gnutls be supported?

Reply:
Not within this pull-request.
We have to build all parts of Gnutls from source then build Gnutls.
But currently all the macOS builds provided by this system since including Gnutls suffer from bug #71

Reply:
So you are suggesting removing gnutls support for the moment?

Reply:
That’s correct, it’s better to have functional wine builds.

At precent if PlayOnMac uses the builds provided by this system they will all the #71 bug. Any wine version built since Gnutls was included. Potentially affected builds start from Wine-3.13 to precent and of course include WineCX

Reply:
Sounds fair. Any idea why is macports gnutls buggy? Can it be because of the symbolic link hack?

Reply:
I'm sure it's related, if I copy out the macport's files were using I did notice the following error during configure stage

`checking for gnutls_cipher_init... no`
Then later
`configure: libgnutls 32-bit development files too old, bcrypt encryption won't be supported.
`
wine will still build but yeah vcrun/dotnet stall.